src/main/java/microsoft/exchange/webservices/data/attribute/Attachable.java

```diff
@@ -31,7 +31,6 @@
 /**
  * The Interface Attachable.
  */
-@Target(ElementType.TYPE)
-@Retention(RetentionPolicy.RUNTIME) public @interface Attachable {
+@Target(ElementType.TYPE) @Retention(RetentionPolicy.RUNTIME) public @interface Attachable {
```

Comment:
I don't like some things in Google's code style guide, but in this point I agree with Erik. It will be better to put annotations one per line. It is commonly used practice.